MANIFEST.in

```diff
@@ -2,3 +2,4 @@ include flybrains/data/*.json
 include flybrains/data/*.csv
 include flybrains/data/*.txt
 include flybrains/meshes/*.ply
+recursive-include flybrains/data *
```

Comment:
That folder contains more than just the transforms. Can you change this to only include .txt and .csv?

Reply:
Resolved